data_steward/resource_files/templates/hpo_report.html

```diff
@@ -233,7 +233,11 @@ <h1>
     </p>
 {% if submission_error is defined %}
     <div class="alert alert-danger">
-        Validation failed for the following reason: {{ submission_error }}.
+        Validation failed for the following reason: {{ submission_error }}.<br>
+        
+        If Your files have not been successfully uploaded to the DRC.
```

Comment:
```suggestion
        Your files have not been successfully uploaded to the DRC.
```

Reply:
I added this because right now, we don't just check all the files are loaded and report that it's failed. so I added an "IF" so that if that's the case they can try running the local_validation.

Reply:
Gotcha. That makes sense. This is good to go then🥳🥳